unit_tests/testInterpolators/testInterpolators.f90

```diff
@@ -0,0 +1,269 @@
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,2.4_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 85.6_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",85.6_SRK
+
+  COMPONENT_TEST('3D Ascending Out Of Range')
+  Interpolant=Interp(label1,label2,label3,table_3D,[0.5_SRK,2.4_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 76.0_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",76.0_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[3.5_SRK,2.4_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 102.0_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",102.0_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,0.5_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 46.0_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",46.0_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,3.5_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 106.0_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",106.0_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,2.4_SRK,0.5_SRK])
+  ASSERT(Interpolant .APPROXEQR. 85.6_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",85.6_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,2.4_SRK,3.5_SRK])
+  ASSERT(Interpolant .APPROXEQR. 85.6_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",85.6_SRK
+  !
+  COMPONENT_TEST('3D Descending In Range')
+  label1(1)=3.0_SRK
+  label1(2)=2.0_SRK
+  label1(3)=1.0_SRK
+  label2(1)=3.0_SRK
+  label2(2)=2.0_SRK
+  label2(3)=1.0_SRK
+  label3(1)=3.0_SRK
+  label3(2)=2.0_SRK
+  label3(3)=1.0_SRK
+  table_3D(1,1,1)=120.0_SRK
+  table_3D(2,1,1)=110.0_SRK
+  table_3D(3,1,1)=100.0_SRK
+  table_3D(1,2,1)=90.0_SRK
+  table_3D(2,2,1)=80.0_SRK
+  table_3D(3,2,1)=60.0_SRK
+  table_3D(1,3,1)=75.0_SRK
+  table_3D(2,3,1)=50.0_SRK
+  table_3D(3,3,1)=40.0_SRK
+  table_3D(1,1,2)=120.0_SRK
+  table_3D(2,1,2)=110.0_SRK
+  table_3D(3,1,2)=100.0_SRK
+  table_3D(1,2,2)=90.0_SRK
+  table_3D(2,2,2)=80.0_SRK
+  table_3D(3,2,2)=60.0_SRK
+  table_3D(1,3,2)=75.0_SRK
+  table_3D(2,3,2)=50.0_SRK
+  table_3D(3,3,2)=40.0_SRK
+  table_3D(1,1,3)=120.0_SRK
+  table_3D(2,1,3)=110.0_SRK
+  table_3D(3,1,3)=100.0_SRK
+  table_3D(1,2,3)=90.0_SRK
+  table_3D(2,2,3)=80.0_SRK
+  table_3D(3,2,3)=60.0_SRK
+  table_3D(1,3,3)=75.0_SRK
+  table_3D(2,3,3)=50.0_SRK
+  table_3D(3,3,3)=40.0_SRK
+  Interpolant=Interp(label1,label2,label3,table_3D,[1.6_SRK,2.4_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 85.6_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",85.6_SRK
+
+  COMPONENT_TEST('3D Descending Out Of Range')
+  Interpolant=Interp(label1,label2,label3,table_3D,[0.5_SRK,2.4_SRK,1.0_SRK])
+  ASSERT(Interpolant .APPROXEQR. 76.0_SRK,'Incorrect Interpolant')
+  FINFO()"Interpolant=", Interpolant, "True value=",76.0_SRK
```

Comment:
in general, we've been moving away from `FINFO` in favor of `ASSERT_EQ`

Reply:
Switched these instances of FINFO over.